Mathlib/NumberTheory/Divisors.lean

```diff
@@ -118,6 +127,25 @@ theorem mem_divisorsAntidiagonal {x : ℕ × ℕ} :
     rw [mul_ne_zero_iff] at hab
     simpa [hab.1, hab.2] using Nat.le_mul_of_pos_right _ hab.2.bot_lt
 
+@[simp]
+lemma divisorsAntidiagonalList_coe {n : ℕ} :
+    n.divisorsAntidiagonalList.toFinset = n.divisorsAntidiagonal := by
+  rw [divisorsAntidiagonalList, divisorsAntidiagonal, List.toFinset_filterMap (f_inj := by aesop),
+    List.toFinset_range'_1_1]
+
+lemma divisorsAntidiagonalList_sorted {n : ℕ} :
```

Comment:
Can you show that the list is `Nodup` as well? It should follow trivially from this